nuget_witsml.md

```diff
@@ -0,0 +1,36 @@
+## Access pre-built Witsml nuget package
+
+If you only need to include the [Witsml](https://github.com/equinor/witsml-explorer/tree/main/Src/Witsml) project
+```csharp
+witsml-explorer/Src/Witsml
+```
```

Comment:
I do not know if the is csharp code explicitly but I get a feeling that it is closer to a shell/path rather than csharp related code. 

```suggestion
```sh
witsml-explorer/Src/Witsml
```
```

Reply:
it is more correct, I agree and I will change it thanks